boards/xtensa/m5stack_core2/m5stack_core2.dts

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/dts-v1/;
+
+#include <espressif/esp32/esp32_common.dtsi>
+#include "m5stack_core2-pinctrl.dtsi"
+#include <zephyr/dt-bindings/display/ili9xxx.h>
+#include <zephyr/dt-bindings/regulator/axp192.h>
+
+/ {
+	model = "esp32";
+	compatible = "espressif,esp32";
+
+	aliases {
+		pwr-led = &pwr_led;
+		uart-0 = &uart0;
+		i2c-0 = &i2c0;
+		watchdog0 = &wdt0;
+		rtc = &pfc8563_rtc;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,flash = &flash0;
+		zephyr,display = &ili9342c;
+		zephyr,keyboard-scan = &kscan_input;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,rtc = &pfc8563_rtc;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <ESP32_CLK_CPU_240M>;
+};
+
+&cpu1 {
+	clock-frequency = <ESP32_CLK_CPU_240M>;
+};
+
+&psram0 {
+	reg = <0x3f800000 DT_SIZE_M(8)>;
+	status = "disabled";
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_tx_gpio1 &uart0_rx_gpio3>;
+	pinctrl-names = "default";
+};
+
+&gpio0 {
+	status = "okay";
+	gpio-reserved-ranges =	<6 11>, // PSRAM
```

Comment:
IIRC, the `gpio-reserved-ranges` attribute is not used in ESP32 Zephyr port as this is handled by the `hal_espressif` macro(s) `SOC_GPIO_VALID_GPIO_MASK` located at the `soc_caps.h`. This macro is used by the Zephyr gpio driver.
Consider to remove the the `gpio-reserved-ranges` from the board dts.

Reply:
Thank you. You are right.
Has been fixed